easybuild/easyconfigs/s/snpEff/snpEff-5.2f-GCCcore-13.2.0-Java-23.0.2.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'Tarball'
+
+name = 'snpEff'
+version = '5.2f'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://pcingola.github.io/SnpEff/'
+description = """SnpEff is a variant annotation and effect prediction tool.
+ It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://snpeff.blob.core.windows.net/versions/']
+sources = ['%%(name)s_v%s_core.zip' % version.replace('.', '_')]
+checksums = ['445f4b73a723a018afbcbcbfe49c77c6771745f55cf662246bf265f53342d961']
+
+dependencies = [
+    # ignore website claim that Java 12+ is required, nothing is compiled for
+    # anything newer than Java 11
+    ('Java', '23', '', SYSTEM),
```

Comment:
Across the `2023b` TC, it seems 11 and 17 are the used versions.
Any reason on jumping here to 23?

Also i think we should stick to LTS versions eg: 11, 17, 21  (https://www.oracle.com/java/technologies/java-se-support-roadmap.html)
unless we really need to do otherwise

Reply:
This issue indicate that we need at least Java 21:

https://github.com/pcingola/SnpEff/issues/565